interface/super/edit_globals.php

```diff
@@ -373,8 +373,13 @@ function checkBackgroundServices()
     'help_file_name' => ""
 );
 $oemr_ui = new OemrUI($arrOeUiSettings);
+$serverConfig = new \OpenEMR\FHIR\Config\ServerConfig();
+$apiUrl = $serverConfig->getBaseApiUrl();
```

Comment:
getting the full url including server may not always work if the global is not set correctly. Can't we just use the relative path here like we normally do when calling scripts (then don't need to worry if the global is set correctly)?

Reply:
I think the issue here is that I'm hitting the API endpoints which all require the oauth global to be setup. All of our apis utilize that oauth global and if anyone has done any kind of proxy on the api and setup the url to be different than the local relative server url then we'd have problems hitting the api.  Since we have a global that could be different then the local setup, I need to make sure if I'm hitting the api that we are hitting the configured endpoint for the api.

I'd rather that we didn't have the global at all, but since we have the global for the api I think we need to use it when communicating with the api.

Reply:
Resolved by using local api.